lib/puppet/defaults.rb

```diff
@@ -1517,8 +1517,7 @@ def self.default_diffargs
     },
     :lastrunreport =>  {
       :default  => "$statedir/last_run_report.yaml",
-      :type     => :file,
-      :mode     => "0640",
+      :type     => :path,
```

Comment:
It's not clear how multiple paths should be used. Does this mean it saves to multiple locations? When reading the lastrunreport, which location should it use?